vulnerabilities/data_source.py

```diff
@@ -228,6 +228,26 @@ def batch_advisories(self, advisories: List[Advisory]) -> Set[Advisory]:
             b, advisories = advisories[: self.batch_size], advisories[self.batch_size :]
             yield b
 
+    @staticmethod
+    def is_cve(id: str):
+        c = id.split("-")
+        if len(c) == 3 and c[0].lower() == "cve" and c[1].isdigit() and c[2].isdigit():
+            return True
+        return False
+
+    @staticmethod
+    def is_vulcoid(id: str):
+        c = id.split("-")
+        if (
+            len(c) == 4
+            and c[0].lower() == "vulcoid"
+            and c[1].isdigit()
+            and c[2].isdigit()
+            and c[3].isdigit()
+        ):
+            return True
+        return False
+
```

Comment:
These are not required. `is-cve`  is duplicated we already a method for doing that validation at https://github.com/nexB/vulnerablecode/blob/3d66b4e82ee31422ab907d3388f739b768ffd2ac/vulnerabilities/helpers.py#L81

`is_vulcoid` is not needed since `vulcoids` are not assigned manually. The advisories without vulnerability id are assigned vulcoid, see https://github.com/nexB/vulnerablecode/blob/3d66b4e82ee31422ab907d3388f739b768ffd2ac/vulnerabilities/models.py#L65